surveys/survey-validator/pubspec.yaml

```
@@ -0,0 +1,15 @@
+name: survey_validator
+description: A sample command-line application.
+version: 1.0.0
+# repository: https://github.com/my_org/my_repo
```

To prevent accidental publishing:

```suggestion
publish_to: none
```